proto/Networking/Requests/Messages/UpgradePokemonMessage.proto

```diff
@@ -2,6 +2,6 @@ syntax = "proto3";
 package POGOProtos.Networking.Requests.Messages;
 
 message UpgradePokemonMessage {
-	fixed64 pokemon_id = 1;
```

Comment:
that was correct

Reply:
https://github.com/Furtif/POGOProtos/blob/master/base/v0.131.1.proto#L10285 dump as out uint64

Reply:
fixed at https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/2699be05eaf2516680c0121818fc42aa5a287daf#diff-e28782438b7ce8a55c71736f98731c1dR5